smtp-user-enum

```diff
@@ -433,6 +434,15 @@ Default: Nothing wrapped""",
         type=str,
         help="MAIL FROM email address. Only used in RCPT mode" + "\nDefault: " + DEFAULT_MAIL_FROM,
     )
+    parser.add_argument(
+        "-l",
+        "--helo",
+        metavar="addr",
+        required=False,
+        default=DEFAULT_HELO,
+        type=str,
+        help="Domain name of sending host used in HELO command."
```

Comment:
Can you add the `+ "\nDefault: " + DEFAULT_HELO` as it was done in line 435

Reply:
Sure thing! Commited few seconds ago